pa11ycrawler/templates/index.html

```diff
@@ -36,8 +36,8 @@ <h2 class="hd-4">Pages Checked</h2>
 
 <%def name="makerow(page)">
     <tr>
-        <td><a href="${page['url']}">${page['page_title']}</a></td>
-        <td><a href="${page['filename']}">Results<span class='sr-only'>&nbsp;for ${page['page_title']}</span></a></td>
+        <td><a href="${page['filename']}">${page['page_title']}</a></td>
+        <td><a href="${page['url']}">View Live</a></td>
```

Comment:
For accessibility purposes, you don't want to have non-unique links, so let's add some offscreen text for this:
```View <span class='sr-only'>${page['page_title']}&nbsp;</span>Live```